build/prow/e2e/Dockerfile

```diff
@@ -27,7 +27,12 @@ FROM ${GOLANG_IMAGE} as builder
 
 WORKDIR /workspace
 
-COPY . .
```

Comment:
This seems to be missed before. I don't think we need to copy any source to the `builder` image.

Reply:
The source is used to build both nomos and junit-report. Both might hypothetically change on any given commit.

Reply:
Oh, right, `junit-report` postprocesses the junit report: http://b/260274932, but I think that rarely changes.

We can still keep `nomos` outside of the builder image.